iot_pkcs11_psa.c

```diff
@@ -2173,7 +2173,7 @@ static CK_RV  FindKeyObjects ( P11SessionPtr_t pxSession,
             /* Import the key into the PKCS#11 context. */
             PKCS11PSAContextImportObject( pxSession->pxFindObjectLabel,
                                           strlen( ( const char * ) pxSession->pxFindObjectLabel ),
-                                          uxPsaDeviceKeyHandle );
+                                          xKeyId );
```

Comment:
Since the `uxPsaDeviceKeyHandle` is not used anymore as part of this function, do we still need to call `PKCS11PSAGetKeyHandle` function to fetch the key handle? According to the comment on this function call (line `2160`) it is used to check if the object was previously stored, can we check if we can find another alternative function to be used to check if the object was previously stored using the `xKeyId`?

Thanks!

Reply:
Indeed, there is cleanup needed. However, `PKCS11PSAGetKeyHandle` uses `P11KeyConfig` to return key handle which is why I decided not to remove it yet. When we start looking to supporting PSA Crypto 1.1, we'll update this.